Header header
bool lower_current
bool request_position
bool spin_motor
float32 velocity
float32 acceleration
int64 position
